docusaurus.config.js

```diff
@@ -183,6 +183,14 @@ module.exports = {
             spec: 'https://raw.githubusercontent.com/transcom/mymove/main/swagger/prime.yaml',
             route: '/api/prime',
           },
+          {
+            spec: 'https://raw.githubusercontent.com/transcom/mymove/main/swagger/prime.yaml',
+            route: '/api/prime/v1',
```

Comment:
Suggestion: You added this `/api/prime/v1` route, but left the prime route that references the same yaml file. You are referencing the prime route and not your new route. Should we just have one route for the `prime.yaml` file?

Reply:
My thought process was that it would be nice if we had the same pattern for the URLs. Since we now have a /v2 I thought adding a /v1 would be nice. At the same time I didn't want to cause issues for folks that had the existing link bookmarked. I think we can continue to leverage this default url to point to the main version.

Reply:
@ronaktruss That totally makes sense to me. Good thinking